specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -851,6 +851,10 @@
             "$ref": "#/definitions/MetricDimension"
           },
           "description": "List of dimension conditions."
+        },
+        "skipMetricValidation": {
+          "type": "boolean",
+          "description": "Skip the metric validation to prevent deployment failure."
```

Comment:
Please ask Harel for the correct description. this is not to prevent deployment failures, but to allow create an alert rule on a custom metric before it is emitted. this is only for custom metrics

Reply:
thanks, changed.